src/app/shared/components/template/services/instance/template-action.service.ts

```diff
@@ -40,7 +58,7 @@ export class TemplateActionService {
     const res = await this.processActionQueue();
     await this.handleActionsCallback([...unhandledActions], res);
     if (!this.container.parent) {
-      await this.container.templateNavService.handleNavActionsFromChild(actions, this.container);
+      await this.templateNavService.handleNavActionsFromChild(actions, this.container);
     }
```

Comment:
this replaces slightly awkward script that would access some global by calling from the parent component (now instantiated correctly above)